server/utilities/distance-utils.js

```diff
@@ -27,10 +27,16 @@ function degToRad(deg) {
  * Validates to ensure that the given lat/lons are valid.
  */
 function validateInputs(lat1, lon1, lat2, lon2) {
+    if (isNaN(lat1) || isNaN(lat2)) {
+        throw new Error("Input is not a number");
+    }
     if (Math.abs(lat1) > 90 || Math.abs(lat2) > 90 || !lat1 || !lat2) {
         throw new Error("Invalid latitude");
     }
 
+    if (isNaN(lon1) || isNaN(lon2)) {
+        throw new Error("Input is not a number");
```

Comment:
This is a good validation, because isNaN automatically converts the value to a number before testing it, so something like "43" should pass. But, could you add a test that shows that you can pass lat = "43" and lon "130" or something, and have it not throw an error? I think you can do something like .to.not.throw(); to accomplish this.